packages/webapp/src/components/Modals/WaterUsageCalculatorModal/index.jsx

```diff
@@ -205,6 +221,7 @@ const WaterUseDepthCalculator = ({ system, setTotalWaterUsage, totalWaterUsage,
         max={100}
         system={system}
         control={modalState.control}
+        hideLineUnitSeparator={true}
```

Comment:
@wasswarichard 

Based off the discussion today please remove this line all references to this 'hideLineUnitSeperator'